### DISCUSSION TITLE
Transferring domain from deleted team account

### DISCUSSION THREAD

**Discussion by foresthpark** [vercel/community#4376] [Account Issue] [answered]
### Question

Hello,

I have accidentally deleted a team account that holds a domain and would like to transfer it to a different hobby account. Is it possible?

Domain name is rocketpunch.dev 

Thank you.

**Comment by foresthpark** [comment]
And the domain was purchased through vercel

**Comment by amyegan** [comment, marked as answer]
Hey @foresthpark! You should have a transfer request notification. After you accept the transfer, the domain should appear in your personal account on the Domains page.

**Reply by foresthpark** [reply]
Hello @amyegan , I found the link but unfortunately the token expired. Is there a way to regenerate the link? Thank you.

**Reply by amyegan** [reply]
You should have a new transfer request now

**Reply by foresthpark** [reply]
Got it! Thank you so much :)
